backend/src/routes/api/notebooks/index.ts

```diff
@@ -87,11 +87,21 @@ module.exports = async (fastify: KubeFastifyInstance) => {
             name: string;
           };
         }>,
+        reply,
       ) => {
         const { namespace, name } = request.params;
         const data = await sanitizeNotebookForSecurity(fastify, request, request.body);
 
-        return await patchNotebook(fastify, data, namespace, name);
+        return patchNotebook(fastify, data, namespace, name)
+          .then((res) => {
+            return res;
+          })
```

Comment:
FWIW, if you omit `.then` this is what happens. You don't need to do it.

Reply:
agh, i forgot about that, thanks for the tip!